### PR TITLE
Added GATK VariantEval wrapper

### PR DESCRIPTION
**Pull request by fgvieira**

### CODE REVIEW

File: bio/gatk/varianteval/wrapper.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+extra = snakemake.params.get("extra", "")
+java_opts = get_java_opts(snakemake)
+
+
+vcf = list(map("--eval {}".format, snakemake.input.vcf))
+
+bam = snakemake.input.get("bam", "")
+if bam:
+    bam = list(map("--input {}".format, bam))
```
Comment (jafors) [inline bio/gatk/varianteval/wrapper.py:17]: Maybe we should allow `input.vcf` and `input.bam` to be strings if only one sample is used. You could check if they are strings or lists and act accordingly. Sth like

```
vcf = snakemake.input.vcf
if isinstance(vcf, str):
    vcf = "--eval  {}".format(vcf)
else:
    vcf = list(map("--eval {}".format, vcf))
```

File: bio/gatk/varianteval/wrapper.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
```
Comment (jafors) [inline bio/gatk/varianteval/wrapper.py:18]: Maybe we should also add a `--comp` input for comparison vcfs, similar to the bam input.
See [here](https://gatk.broadinstitute.org/hc/en-us/articles/360040507171-VariantEval-BETA-#--comparison)

File: bio/gatk/varianteval/test/Snakefile [hunk 3]
```diff
@@ -0,0 +1,16 @@
+rule gatk_varianteval:
+    input:
+        vcf=["calls/snvs.vcf"],
```
Comment (jafors) [inline bio/gatk/varianteval/test/Snakefile:3]: As mentioned in the `wrapper.py`, I feel this should be able to be a string in single sample runs.

**Review by jafors — CHANGES_REQUESTED** [review]
Thanks for the addition! I have added some remarks, feel free to comment if you disagree!

**Review by johanneskoester — APPROVED** [review]

**Pull request merged**